server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DLPConfigurationRoutesTest.java

```diff
@@ -164,51 +134,130 @@ void setup() throws Exception {
 
     @Nested
     class DefineStore {
+
         @Test
-        void putShouldstoreTheConfigurations() {
-            String body = JsonNodeFactory.instance.arrayNode()
-                .add(JSON_WITH_POSITIVE_TARGETS)
-                .add(JSON_WITH_NEGATIVE_TARGETS)
-                .toString();
+        void putShouldStoreTheConfigurations() {
+            String storeBody =
+                "[{" +
+                "  \"id\": \"1\"," +
```

Comment:
IMO the client should not be allowed to specify an ID.

The James server has to set it himself.

Reply:
I did asked you about ID generating:
> If ID should be generated in James, then it's better to generate ID inside`DLPConfigurationStore`, not in webadmin, because it's easy to generate appropriate ID based on  `DLPConfigurationStore` implementation (eventsouring, cassandra, memory...) WDYT?

Reply:
Sure.
 - What is the format of ID?
 - Are duplicated ids allowed?
 - What value makes sens as an Id?

I think a user don't know / should not know about all of this. Ids should be backend assigned IMO. I saw also comments on this sens on the chat. Team, WDYT?

Reply:
We will finally take Ids from the client